app/scripts/migrations/index.js

```diff
@@ -185,6 +188,9 @@ const migrations = [
   m088,
   m089,
   m090,
+  ///: BEGIN:ONLY_INCLUDE_IN(blockaid)
+  m091,
+  ///: END:ONLY_INCLUDE_IN
```

Comment:
as we have discussed, I am not sure that this is possible as it would could block or prevent subsequent migrations from running or if subsequent were executed before this one, then this one would never be executed. Example:
1. migration 91 only runs in Flask
2. a subsequent PR adds migration 92 and expects it to run everywhere (Flask, Beta, Main)
3. remove code fencing for blockaid and promote it to Main. At this time the versioned data is on 92. But on main it didn't run 91.. and it won't run it, because it will filter pending migrations to run with this [validation](https://github.com/MetaMask/metamask-extension/blob/develop/app/scripts/lib/migrator/index.js#L77-L80). Aaand `versionedData.meta.version` is going to be 92.

The only way that I see it is to reference both `securityAlertsEnabled` and `transactionSecurityCheckEnabled` properties in Flask, but then we would do this migration once we were ready to promote to main.

@segun and @jpuri what do you think about it?